ProcessMaker/processmaker
# Implement Creation of Screen Templates Database Table and Models

This PR implements the creation of the `screen_templates` database table along with additional creations of the screen template model and screen template interface. These changes mark the initial steps in developing screen templates.

## How to Test

1. Ensure you have database access in your testing environment.
2. Run `php artisan migrate` in your local environment's terminal.
3. Observe the new `screen_templates` table in your database.

## Related Tickets & Packages
- [ FOUR-13475](https://processmaker.atlassian.net/browse/FOUR-13475)

## Code Review Checklist
